editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -456,6 +456,10 @@ void SpriteFramesEditor::_animation_select() {
 	if (updating)
 		return;
 
+	double value = anim_speed->get_line_edit()->get_text().to_double();
+	if (value != frames->get_animation_speed(edited_anim))
```

Comment:
I would use `!Math::is_equal_approx(value, rames->get_animation_speed(edited_anim))` instead as the current check is likely to fail most of the time due to precision issues.